schedule/models/events.py

```diff
@@ -637,13 +637,12 @@ def hours(self):
     def get_absolute_url(self):
         if self.pk is not None:
             return reverse(
-                "occurrence",
-                kwargs={"occurrence_id": self.pk, "event_id": self.event.id},
+                "occurrence", kwargs={"occurrence_id": self.pk, "event_id": self.event_id}
```

Comment:
Django caches foreignkey ids, so doing this means one less db read. [read more](https://django.doctor/advice/C9001)